radionets/dl_framework/loss_functions.py

```diff
@@ -204,14 +230,10 @@ def comb_likelihood(x, y):
     y_phase = y[:, 1]
 
     loss_amp = (
-        0.5 * torch.log(amp_unc.pow(2))
-        + ((y_amp - amp_pred).pow(2) / amp_unc.pow(2))
-        + (y_amp - amp_pred)
+        2 * torch.log(amp_unc) + ((y_amp - amp_pred).pow(2) / amp_unc.pow(2))
     ).mean()
     loss_phase = (
-        0.5 * torch.log(phase_unc.pow(2))
-        + ((y_phase - phase_pred).pow(2) / phase_unc.pow(2))
-        + (y_phase - phase_pred)
```

Comment:
when did we add the addiotional +(y -pred) term?

Reply:
That was you three months ago [here](https://github.com/radionets-project/radionets/commit/4adbc445f3b0077d0e154ab09b1fd6f50eb463dc#). I do not know why though.

Reply:
Ok, we drop it for now